bcda/models/models.go

```diff
@@ -323,15 +323,26 @@ func GetSuppressedBlueButtonIDs(db *gorm.DB) []string {
 
 	var suppressedBBIDs []string
 
-	db.Raw(`SELECT DISTINCT s.blue_button_id
-			FROM (
-				SELECT blue_button_id, MAX(effective_date) max_date
-				FROM suppressions
-				WHERE effective_date <= NOW() AND preference_indicator != '' AND blue_button_id != '' AND blue_button_id IS NOT NULL
-				GROUP BY blue_button_id
-			) h
-			JOIN suppressions s ON s.blue_button_id = h.blue_button_id and s.effective_date = h.max_date
-			WHERE preference_indicator = 'N'`).Pluck("blue_button_id", &suppressedBBIDs)
+	// Set the window that the suppression query should examine. Will default to 60 if no value is provided
+	var suppressionLookback string
```

Comment:
Should this ben an int?

Reply:
since this value only gets used to put into the raw sql query we would have to convert it to a string anyway so this should probably stay a string

Reply:
Hmm, but the value is an integer, correct?  I think it should exist in isolation... and be typed appropriately... how it is used downstream is up to the user of the variable.